packages/kbn-eslint-plugin-disable/src/rules/no_naked_eslint_disable.test.ts

```diff
@@ -0,0 +1,286 @@
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import dedent from 'dedent';
+import { RuleTester } from 'eslint';
+import { NoNakedESLintDisableRule } from './no_naked_eslint_disable';
+
+const tsTester = [
+  '@typescript-eslint/parser',
+  new RuleTester({
+    parser: require.resolve('@typescript-eslint/parser'),
+    parserOptions: {
+      sourceType: 'module',
+      ecmaVersion: 2018,
+      ecmaFeatures: {
+        jsx: true,
+      },
+    },
+  }),
+] as const;
+
+const babelTester = [
+  '@babel/eslint-parser',
+  new RuleTester({
+    parser: require.resolve('@babel/eslint-parser'),
+    parserOptions: {
+      sourceType: 'module',
+      ecmaVersion: 2018,
+      requireConfigFile: false,
+      babelOptions: {
+        presets: ['@kbn/babel-preset/node_preset'],
+      },
+    },
+  }),
+] as const;
+
+for (const [name, tester] of [tsTester, babelTester]) {
+  describe(name, () => {
+    tester.run('@kbn/disable/no_naked_eslint_disable', NoNakedESLintDisableRule, {
+      valid: [
+        {
+          filename: 'foo.ts',
+          code: dedent`
+            // eslint-disable no-var
+          `,
+        },
+        {
+          filename: 'foo.ts',
+          code: dedent`
+            // eslint-disable-next-line no-use-before-define
+          `,
+        },
+        {
+          filename: 'foo.ts',
+          code: dedent`
+            // eslint-disable-line no-use-before-define
+          `,
+        },
+        {
+          filename: 'foo.ts',
+          code: dedent`
+            /* eslint-disable no-var */
+          `,
+        },
+        {
+          filename: 'foo.ts',
+          code: dedent`
+            /* eslint-disable no-console, no-control-regex*/
+          `,
+        },
+        {
+          filename: 'foo.ts',
+          code: dedent`
+            alert('foo'); // eslint-disable-line no-alert
+          `,
+        },
+        {
+          filename: 'foo.ts',
+          code: dedent`
+            const foo = 'foo';
+            let bar = 'ba';
+            /* eslint-disable no-alert */
+            alert(foo);
+            /* eslint-enable no-alert */
+            bar += 'r';
+          `,
+        },
+        {
+          filename: 'foo.ts',
+          code: dedent`
+            const foo = 'foo';
+            let bar = 'ba';
+            /* eslint-disable-next-line no-alert */
+            alert(foo);
+            bar += 'r';
+          `,
+        },
+        {
+          filename: 'foo.ts',
+          code: dedent`
+            const foo = 'foo';
+            let bar = 'ba';
+            alert(foo);/* eslint-disable-line no-alert */
+            bar += 'r';
+          `,
+        },
+      ],
+
+      invalid: [
+        {
+          filename: 'foo.ts',
+          code: dedent`
+            /* eslint-disable */
+            const a = 1;
+          `,
+          errors: [
+            {
+              line: 1,
+              message:
```

Comment:
Since you used messageId's you can change this to `messageId`: https://eslint.org/docs/latest/developer-guide/nodejs-api#testing-errors-with-messageid

Reply:
nice catch 👍 changed